packages/core/src/getCurrentHubShim.ts

```diff
@@ -22,7 +22,10 @@ import {
  * usage
  */
 // eslint-disable-next-line deprecation/deprecation
-export function getCurrentHubShim(): Hub {
+export function getCurrentHubShim(): Hub & {
+  getStackTop: () => { client: Client | undefined; scope: Scope };
+  isOlderThan: () => boolean;
+} {
```

Comment:
Just noticed: We actually export this publicly from @sentry/core (not sure why we did this at all). Do we care that we widen the return type here?